dd-java-agent/instrumentation/jedis-1.4/src/main/java/datadog/trace/instrumentation/jedis/JedisInstrumentation.java

```diff
@@ -59,6 +60,7 @@ public JedisInstrumentation() {
     @Advice.OnMethodEnter(suppress = Throwable.class)
     public static Scope startSpan(@Advice.Argument(1) final Command command) {
       final Scope scope = GlobalTracer.get().buildSpan("redis.command").startActive(true);
+      scope.span().setOperationName(command.name());
```

Comment:
Why not do that in the `buildSpan()` above? Also, I think it would be more readable for the user if the operation name still said `redis`:

```java
final Scope scope = GlobalTracer.get()
    .buildSpan("redis." + command.name().toLowerString())
    .startActive(true);
```